pycloudflare/utils.py

```diff
@@ -0,0 +1,17 @@
+def _find_server_error(response, err_code=None, msg_part=None):
+    if err_code is None and msg_part is None:
```

Comment:
I would separate those things (different methods/wrappers - msg matching and code matching.  Implementation would be cleaner (and also we don't need msg matching atm)